collectors/python.d.plugin/openldap/openldap.chart.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Description: openldap netdata python.d module
+# Author: Manolis Kartsonakis (ekartsonakis)
+# SPDX-License-Identifier: GPL-3.0+
+
+try: 
+     import ldap 
+     HAS_LDAP = True 
+except ImportError: 
+     HAS_LDAP = False
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+# default module values (can be overridden per job in `config`)
+update_every = 10
+priority = 60000
+retries = 60
+
+ORDER = [
+    'total_connections', 'bytes_sent',
+    'operations', 'referrals_sent',
+    'entries_sent', 'ldap_operations', 'waiters'
+]
+
+CHARTS = {
+  
+     'total_connections': {
+        'options': [None, 'Total Connections', 'connections', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['total_connections', None, 'incremental']
+        ]
+    },
+  
+    'bytes_sent': {
+        'options': [None, 'Bytes Statistics', 'Bytes', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['bytes_sent', None, 'incremental']
+        ]
+    },
+    'operations': {
+        'options': [None, 'Operations', 'ops', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['completed_operations', None, 'incremental'],
+            ['initiated_operations', None, 'incremental']
+        ]
+    },
+    'referrals_sent': {
+        'options': [None, 'Referrals Statistics', 'referalls_sent', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['referrals_sent', None, 'incremental']
+        ]
+    },
+    'entries_sent': {
+        'options': [None, 'Entries Statistics', 'entries_sent', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['entries_sent', None, 'incremental']
+        ]
+    },
+    'ldap_operations': {
+        'options': [None, 'Operations', 'bind_operations', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['bind_operations', None, 'incremental'],
+            ['search_operations', None, 'incremental'],
+            ['unbind_operations', None, 'incremental'],
+            ['add_operations', None, 'incremental'],
+            ['delete_operations', None, 'incremental'],
+            ['modify_operations', None, 'incremental'],
+            ['compare_operations', None, 'incremental']
+        ]
+    },
+    'waiters': {
+        'options': [None, 'Waiters', 'amount', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['write_waiters', None, 'incremental'],
+            ['read_waiters', None, 'incremental']
+        ]
+    },
+}
+
+# Stuff to gather - make tuples of DN dn and attrib to get
+searchlist = {
```

Comment:
1. make it upper case

2. i suggest to change format to

<details><summary>clicke me!</summary>
<p>

```python
SEARCH_LIST = {
    'total_connections': (
        'cn=Total,cn=Connections,cn=Monitor',
        'monitorCounter',
    ),
    'bytes_sent': (
        'cn=Bytes,cn=Statistics,cn=Monitor',
        'monitorCounter',
    ),
    'completed_operations': (
        'cn=Operations,cn=Monitor',
        'monitorOpCompleted',
    ),
    'initiated_operations': (
        'cn=Operations,cn=Monitor',
        'monitorOpInitiated',
    ),
    'referrals_sent': (
        'cn=Referrals,cn=Statistics,cn=Monitor',
        'monitorCounter',
    ),
    'entries_sent': (
        'cn=Entries,cn=Statistics,cn=Monitor',
        'monitorCounter',
    ),
    'bind_operations': (
        'cn=Bind,cn=Operations,cn=Monitor',
        'monitorOpCompleted',
    ),
    'unbind_operations': (
        'cn=Unbind,cn=Operations,cn=Monitor',
        'monitorOpCompleted',
    ),
    'add_operations': (
        'cn=Add,cn=Operations,cn=Monitor',
        'monitorOpInitiated',
    ),
    'delete_operations':  (
        'cn=Delete,cn=Operations,cn=Monitor',
        'monitorOpCompleted',
    ),
    'modify_operations': (
        'cn=Modify,cn=Operations,cn=Monitor',
        'monitorOpCompleted',
    ),
    'compare_operations': (
        'cn=Compare,cn=Operations,cn=Monitor',
        'monitorOpCompleted',
    ),
    'search_operations': (
        'cn=Search,cn=Operations,cn=Monitor',
        'monitorOpCompleted',
    ),
    'write_waiters': (
        'cn=Write,cn=Waiters,cn=Monitor',
        'monitorCounter',
    ),
    'read_waiters': (
        'cn=Read,cn=Waiters,cn=Monitor',
        'monitorCounter',
    ),
}
```
</p>
</details>

Reply:
I don't think that this change increase readability of the code despite the fact that it is more pythonic.  I tryed something in between.

```
SEARCH_LIST = {
    'total_connections': (
        'cn=Total,cn=Connections,cn=Monitor', 'monitorCounter',
    ),
    'bytes_sent': (
        'cn=Bytes,cn=Statistics,cn=Monitor', 'monitorCounter',
    ),
```

Reply:
fixed